octoprint_mrbeam/__init__.py

```diff
@@ -1300,6 +1293,16 @@ def custom_materials(self, data):
         # self._logger.info("custom_material(): response: %s", data)
         return make_response(jsonify(res), 200)
 
+    def get_laser_cutter_mode(self):
+        # TODO: SW-3719 return actual laser cutter mode
+        return "default"
+
+    def get_laser_head_model(self):
+        return self.laserhead_handler.get_current_used_lh_data()["model"]
+
+    def get_laser_head_serial(self):
+        return self.laserhead_handler.get_current_used_lh_data()["serial"]
```

Comment:
I think these 2 methods should be get_current_laser_head... as there is more than 1 laser head model and serial (current and old)

Reply:
Agree, I'll change it